packages/babel-plugin-transform-spread/test/fixtures/call-context/parenthesized-expressions/output.js

```diff
@@ -0,0 +1,3 @@
+var _a;
+
+((_a = a).b.apply)(_a, babelHelpers.toConsumableArray(args));
```

Comment:
It would be nice if this was transpiled to
```js
((_a = a).b).apply(_a, babelHelpers.toConsumableArray(args));
```

Reply:
That's a lot nicer, I made it happen 🙌